modules/DynamicFields/templates/Fields/Forms/int.tpl

```diff
@@ -132,6 +132,6 @@ formsWithFieldLogic=null;
     </td>
 </tr>
 <script>
-	formsWithFieldLogic=new addToValidateFieldLogic('popup_form_id', 'int_min', 'int_max', 'int_default', 'int_len', 'int', 'Invalid Logic.');
+	formsWithFieldLogic=new addToValidateFieldLogic('popup_form_id', 'int_min', 'int_max', 'int_default', 'int_len', 'int', '{sugar_translate module="DynamicFields" label="LBL_ERR_LOGIC_INTEGER"}');
```

Comment:
Hi @clemente-raposo,
thanks for questions, I'll try my best to answer, not sure I'm following. 

1) Message is not the case, we can put there the same 'Logic error'. We do want the application to be more agile for translation purpoces and in this case the idea of changes was to substitute hard coded value with label to avoid something like this: 

![image](https://github.com/salesagility/SuiteCRM/assets/116086008/2df41616-f90b-4eed-921a-0c835c9541f4)

2) In that case the error was that default value for the field is out of the range, on screen above another example 
we have range from 100 to 200 and at the same time we are trying to set a default value for that field less than the innitial value of the range. 

Many thanks. 

Regards,
Serhii